build_tools/logo.py

```diff
@@ -34,6 +34,11 @@ def convert_svg_to_image(
 
         # Determine the dimensions for a square based on the cropped image
         max_dim = max(img_cropped.size)
+
+        if padding:
+            padding = int(padding) / 100
```

Comment:
None blocking, but I  wonder if we could add some additional error handling to ensure that the padding value is a valid percentage (i.e. between 0 and 100). i.e. ensure that the padding value is within this range, and raising an error if it is not.

Reply:
We definitely could - as this was developer only code, I was not as thorough.